andela/ah-frontend-realers
# #166187815 User is able to Signup

#### What does this PR do?

Enables users to signup

#### Description of task completed
* created actions and reducers relevant for signing up a user

#### How should this be manually tested on your local machine
* Do a `git fetch origin ft-signup-166187815`
* Do a `git checkout ft-signup-166187815`
* Run this command `$ npm install`
* Run this command `$ npm start`
* you should be able to view a simple descriptive form that enables you to signup.



finishes [#166187815](https://www.pivotaltracker.com/n/projects/2350340/stories/166187815)
